### PR TITLE
Chore: Update index.scss to include gutenberg blocks library CSS

### PR DESCRIPTION
**Pull request by derekshirk**
## Overview

See #789, and Tyler's suggestion [here](https://github.com/cloudfour/cloudfour.com-patterns/issues/789#issuecomment-642322457). 

In order to remedy a CSS source order loading issue that was causing the WordPress Gutenberg block library CSS from trumping our patterns CSS in Storybook previews, this PR does the following: 

• Creates a new `index-with-wp-block-library` file that imports WP styles before our pattern styles (index.scss)
• Updates `.storybook/preview.js` to include the new `index-with-wp-block-library` instead of `index`
• Removes `@wordpress/block-library/build-style/style.css` from `src/vendor/wordpress/gutenberg.stories.mdx`, previously added in #778 

## Consideration 

With #778 (WP) Gutenberg block styles were only loaded on the Gutenberg stories. With this update, as I understand it, the Gutenberg CSS will now be loaded on every Storybook page. If that is a concern we should explore other alternative approaches. 

### Before
Notice the secondary button is outlined with a black border and black text. 

Screen Shot 2020-06-16 at 3 08 54 PM

### After
Here, the secondary button is styled as expected with our `c-button c-button--secondary` styles.

Screen Shot 2020-06-16 at 2 22 30 PM

## Testing

1. View [deploy preview](https://deploy-preview-798--cloudfour-patterns.netlify.app/?path=/story/vendor-wordpress-gutenberg--core-button)
1. Confirm the secondary button is outlined with a blue border and has blue text. 

---

See #789

### CODE REVIEW

**Comment by derekshirk** [comment]
> If these changes should be published to npm, you need to add a changeset.

I am unsure if this PR requires a changeset. 🤔

File: .storybook/preview.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import * as breakpoints from '../src/design-tokens/breakpoint.yml';
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { ratio } from '../src/design-tokens/modular-scale.yml';
 import 'focus-visible';
-import '../src/index.scss';
+import '../src/index-with-wp-block-library.scss';
```
Comment (derekshirk) [inline .storybook/preview.js:11]: I am unsure if this file name is laughable or not. I am open to suggestions here.

Comment (tylersticka) [reply .storybook/preview.js:11]: Maybe `index-with-dependencies` in case we want to add something else later?

Comment (derekshirk) [reply .storybook/preview.js]: I like that suggestion. Thank you. Updated.

**Pull request merged**